treeherder/model/models.py

```diff
@@ -1251,6 +1251,23 @@ def elastic_search_insert(self):
             return es_line
 
 
+class Group(models.Model):
+    """
+    The test harness Group.  Sometimes this is a Manifest, but not always.
+    Not to be confused with JobGroup which is Treeherder specific.
+    """
+    id = models.BigAutoField(primary_key=True)
+    name = models.CharField(max_length=200, unique=True)
```

Comment:
Where does 200 come from? I guess that windows path length limits might impose some natural limits on the length if this is always a path.